### PR TITLE
PMM-6395 VMagent integration

### PR DESCRIPTION
**Pull request by f41gh7**
https://github.com/Percona-Lab/pmm-submodules/pull/1185
~https://github.com/percona/pmm/pull/603~
https://github.com/percona/pmm-admin/pull/122
https://github.com/percona/pmm-agent/pull/201
https://github.com/percona/pmm-managed/pull/501
https://github.com/percona/grafana-dashboards/pull/1034
https://github.com/Percona-Lab/pmm-api-tests/pull/140

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-fdf5e04
client docker - perconalab/pmm-client-fb:PR-1185-fdf5e04
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-fdf5e04.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-4ad5654
client docker - perconalab/pmm-client-fb:PR-1185-4ad5654
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-4ad5654.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4629/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4011/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5217/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-a748c45
client docker - perconalab/pmm-client-fb:PR-1185-a748c45
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-a748c45.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4666/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4048/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5262/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-8067a7c
client docker - perconalab/pmm-client-fb:PR-1185-8067a7c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-8067a7c.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4698/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4079/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5296/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-0e282a4
client docker - perconalab/pmm-client-fb:PR-1185-0e282a4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-0e282a4.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4702/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4083/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5302/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-56d436b
client docker - perconalab/pmm-client-fb:PR-1185-56d436b
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-56d436b.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-81d9223
client docker - perconalab/pmm-client-fb:PR-1185-81d9223
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-81d9223.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-81d9223
client docker - perconalab/pmm-client-fb:PR-1185-81d9223
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-81d9223.tar.gz

File: .gitmodules [hunk 1]
```diff
@@ -62,7 +62,7 @@
 [submodule "qan-api2"]
 	path = sources/qan-api2/src/github.com/percona/qan-api2
 	url = https://github.com/percona/qan-api2
-	branch = master
+	branch = release/2.12
```
Comment (BupycHuk) [inline .gitmodules:65]: let's replace to `master`

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-81d9223
client docker - perconalab/pmm-client-fb:PR-1185-81d9223
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-81d9223.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-d01a3c8
client docker - perconalab/pmm-client-fb:PR-1185-d01a3c8
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-d01a3c8.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-85fff9c
client docker - perconalab/pmm-client-fb:PR-1185-85fff9c
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-85fff9c.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-55a96d0
client docker - perconalab/pmm-client-fb:PR-1185-55a96d0
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-55a96d0.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-963e262
client docker - perconalab/pmm-client-fb:PR-1185-963e262
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-963e262.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-a3cef47
client docker - perconalab/pmm-client-fb:PR-1185-a3cef47
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-a3cef47.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-fa53ac8
client docker - perconalab/pmm-client-fb:PR-1185-fa53ac8
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-fa53ac8.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-a2c7946
client docker - perconalab/pmm-client-fb:PR-1185-a2c7946
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-a2c7946.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-a87a8a4
client docker - perconalab/pmm-client-fb:PR-1185-a87a8a4
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-a87a8a4.tar.gz

**Comment by AlekSi** [comment]
https://github.com/Percona-Lab/pmm-submodules/pull/1173 merged, please re-create that one

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-b5615b2
client docker - perconalab/pmm-client-fb:PR-1185-b5615b2
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-b5615b2.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-b5615b2&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-b5615b2.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-b5615b2
client docker - perconalab/pmm-client-fb:PR-1185-b5615b2
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-b5615b2.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-b5615b2&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-b5615b2.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-b5615b2
client docker - perconalab/pmm-client-fb:PR-1185-b5615b2
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-b5615b2.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-b5615b2&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-b5615b2.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4857/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4239/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5512/

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4857/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4239/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5512/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-ef45c87
client docker - perconalab/pmm-client-fb:PR-1185-ef45c87
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-ef45c87.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-ef45c87&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-ef45c87.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-e817a08
client docker - perconalab/pmm-client-fb:PR-1185-e817a08
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-e817a08.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-e817a08&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-e817a08.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-1b731a2
client docker - perconalab/pmm-client-fb:PR-1185-1b731a2
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-1b731a2.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-1b731a2&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-1b731a2.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-952f558
client docker - perconalab/pmm-client-fb:PR-1185-952f558
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-952f558.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-952f558&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-952f558.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4869/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4251/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5530/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-5d40a52
client docker - perconalab/pmm-client-fb:PR-1185-5d40a52
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-5d40a52.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-5d40a52&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-5d40a52.tar.gz

File: build/bin/vars [hunk 2]
```diff
@@ -42,4 +42,5 @@ source_tarball=${root_dir}/results/source_tarball/pmm2-client-${pmm_version}.tar
 binary_tarball=${root_dir}/results/tarball/pmm2-client-${pmm_version}.tar.gz
 
 # https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/pmm-6401-v1.44.0
-vmagent_commit_hash=4dc13754d882fe2d6b4ba446b21febbea12b1f73
+#vmagent_commit_hash=4dc13754d882fe2d6b4ba446b21febbea12b1f73
+vmagent_commit_hash=v1.46.0
```
Comment (AlekSi) [inline build/bin/vars:46]: We really want to use commit hash there, not tag, because tag can be moved. And I guess that should be pmm-6401-v1.46.0?

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-5176a5a
client docker - perconalab/pmm-client-fb:PR-1185-5176a5a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-5176a5a.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-5176a5a&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-5176a5a.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-5176a5a
client docker - perconalab/pmm-client-fb:PR-1185-5176a5a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-5176a5a.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-5176a5a&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-5176a5a.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-5176a5a
client docker - perconalab/pmm-client-fb:PR-1185-5176a5a
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-5176a5a.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-5176a5a&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-5176a5a.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4880/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4262/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5543/

File: build/bin/vars (hunk 2 above)
Comment (f41gh7) [reply build/bin/vars:46]: get it, will use commit hash. There is no need for tag `pmm-6401-v1.46.0` here,  `vmagent` can be build from upstream version.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-66b7a4f
client docker - perconalab/pmm-client-fb:PR-1185-66b7a4f
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-66b7a4f.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-66b7a4f&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-66b7a4f.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-671111d
client docker - perconalab/pmm-client-fb:PR-1185-671111d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-671111d.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-671111d&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-671111d.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4881/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4263/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5544/

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4883/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4265/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5546/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-fc5a7c5
client docker - perconalab/pmm-client-fb:PR-1185-fc5a7c5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-fc5a7c5.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-fc5a7c5&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-fc5a7c5.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4885/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4267/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5549/

File: build/pmm-client-docker/Dockerfile [hunk 3]
```diff
@@ -12,7 +12,7 @@ RUN install -d -o pmm-agent -g pmm-agent -m 0775 /usr/local/percona/pmm2
 
 RUN tar -zxpf /tmp/pmm2-client.tar.gz -C /tmp \
     && cd /tmp/pmm2-client-* \
-    && env PMM_USER=pmm-agent PMM_GROUP=pmm-agent ./install_tarball \
+    && env PMM_USER=pmm-agent PMM_GROUP=root ./install_tarball \
```
Comment (AlekSi) [inline build/pmm-client-docker/Dockerfile:15]: Cherry-picked from https://github.com/Percona-Lab/pmm-submodules/pull/1233 (already merged into PMM-2.0), can be reset during the next PMM-2.0 reset

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-bb09167
client docker - perconalab/pmm-client-fb:PR-1185-bb09167
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-bb09167.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-bb09167&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-bb09167.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-7beb9d6
client docker - perconalab/pmm-client-fb:PR-1185-7beb9d6
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-7beb9d6.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-7beb9d6&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-7beb9d6.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4888/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4270/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5553/

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4889/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4271/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5554/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-f827414
client docker - perconalab/pmm-client-fb:PR-1185-f827414
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-f827414.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-f827414&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-f827414.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-51e0f81
client docker - perconalab/pmm-client-fb:PR-1185-51e0f81
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-51e0f81.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-51e0f81&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-51e0f81.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4891/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4273/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5556/

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4893/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4275/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5558/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1185-51bd306
client docker - perconalab/pmm-client-fb:PR-1185-51bd306
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-51bd306.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1185-51bd306&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1185-51bd306.tar.gz

**Comment by JNKPercona** [comment]
Some Tests have Failed Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/4902/ BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/4284/ & UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/5569/

**Pull request closed without merging**